### PR TITLE
feat(user): change vuex page store to pinia

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
- change vuex page store to pinia
- update pages

### CODE REVIEW

File: src/services/administration/store/user-page-store.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        timezone: computed(() => store.state.user.timezone || 'UTC'),
+        selectedUsers: computed(() => {
+            const users = [] as User[];
+            state.selectedIndices.map((d) => users.push(state.users[d]));
+            return users;
+        }) || [],
+    });
+
+    /* Actions */
+    const _getArrayWithNotDuplicatedItem = (array) => [...new Set(array)];
```
Comment (sulmoJ) [inline src/services/administration/store/user-page-store.ts:38]: Question: Last time I used `_`, the warning log came up on the console, is it okay this time??

Comment (sulmoJ) [reply src/services/administration/store/user-page-store.ts:38]: Personally, I really like the underbar prefix in the internal function!

**Review by sulmoJ — APPROVED** [review]
UserList action exists, but it has not been used.
Where is it used?
LGTM!!

File: src/services/administration/store/user-page-store.ts (hunk 1 above)
Comment (yuda110) [reply src/services/administration/store/user-page-store.ts:38]: It seems to give such a warning when using a `_state` in a template.
There is no warning yet for using within a script!

**Pull request merged**